src/tools/rustfmt/src/config/options.rs

```diff
@@ -142,13 +142,6 @@ pub enum HexLiteralCase {
     Lower,
 }
 
-#[config_type]
-pub enum ReportTactic {
-    Always,
-    Unnumbered,
-    Never,
-}
```

Comment:
Let's add `﻿#[allow(unused_imports)]` instead if the maintainer is asking.

Reply:
This `use` statement is generated by the proc macro, which means we should use ﻿`#![allow(unused_imports)]` for this module.